var/spack/repos/builtin/packages/last/package.py

```diff
@@ -16,10 +16,13 @@ class Last(MakefilePackage):
     git = "https://gitlab.com/mcfrith/last.git"
     maintainers("snehring")
 
+    version("1499", commit="2cc68d3ba8ae5ca46ceeb69302aef18b9db04f46")
     version("1282", commit="4368be912f4759e52b549940276f1adf087f489a")
     version("869", sha256="6371a6282bc1bb02a5e5013cc463625f2ce3e7746ff2ea0bdf9fe6b15605a67c")
 
```

Comment:
Requested change:

Add a `depends_on("zlib-api")` because if the host does not have zlib development installed, the build fails:
```suggestion
   depends_on("zlib-api")

```
You can commit it by clicking the button "Commit suggestion".

I already tested it but could not push it because the write permission for maintainers to the branch was disabled in the PR or your repo.

Reply:
I'll have to examine why the zlib headers have snuck into my environment. Thanks for catching this.